ciao-image/main.go

```diff
@@ -31,6 +32,8 @@ var identity = "https://localhost:35357/"
 var userName = "csr"
 var password = "hello"
 var mountPoint = "/var/lib/ciao/images"
+var dbDir = "/var/lib/ciao/images"
```

Comment:
I think it's probably best not to store the db in this directory, which up to now has just been used for actual images.  How about /var/lib/ciao/ciao-image/ciao-image.db